docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While data frames support indexing syntax that is very similar to
```

Comment:
The text seems to say that this is different from matrices. Actually this is exactly the same - normally indexing into a data frame creates a copy like for matrices.

Reply:
OH! Sorry, moving back and forth between numpy and julia too much these days...